Dockerfile

```diff
@@ -1,7 +1,15 @@
 FROM golang:1.14.2-buster as builder
+RUN apt-get update && \
+    apt-get install -y zip && \
+    mkdir protoc && \
+    wget -O protoc/protoc.zip https://github.com/protocolbuffers/protobuf/releases/download/v3.12.3/protoc-3.12.3-linux-x86_64.zip && \
+    unzip protoc/protoc.zip -d protoc/ && \
+    mv protoc/bin/protoc /usr/local/bin/ && \
+    go get github.com/golang/protobuf/protoc-gen-go && \
+    curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.27.0
```

Comment:
I'm wondering if we want to test even at build time. Then I thought we don't build that often, so it might be still good to do it? In case someone just push to master?

Reply:
Yeah and we never test the docker image so it was easy enough to do here.